cfme/middleware/__init__.py

```diff
@@ -53,3 +68,63 @@ def download(extension):
         download_btn("Download as {}".format(extensions_mapping[extension]))
     except:
         raise ValueError("Unknown extention. check the extentions_mapping")
+
+
+def add_deployment(filename, runtime_name=None, enable_deploy=True, cancel=False):
```

Comment:
Same with the underlying functions, they seems to be fitting as methods for Deployment, but I can just be wrong, I am not that familiar with middleware UI.